assets/js/src/newsletters/types.jsx

```diff
@@ -28,14 +28,17 @@ class NewsletterTypes extends React.Component {
 
     return _.map(window.mailpoet_automatic_emails, (automaticEmail) => {
       const email = automaticEmail;
-      const onClick = _.partial(this.setupNewsletter, automaticEmail.slug);
+      const onClick = window.mailpoet_premium_active
```

Comment:
Why do we need this change? Does the disabled link trigger `onClick`?

Reply:
Yes, it does!